src/utils/index.js

```diff
@@ -25,19 +25,15 @@ export function getDataTransferItems(event) {
 
 // Firefox versions prior to 53 return a bogus MIME type for every file drag, so dragovers with
 // that MIME type will always be accepted
-export function fileAccepted(file, accept) {
-  return file.type === 'application/x-moz-file' || accepts(file, accept)
-}
+export const fileAccepted = (file, accept) =>
+  file.type === 'application/x-moz-file' || accepts(file, accept)
 
-export function fileMatchSize(file, maxSize, minSize) {
-  return file.size <= maxSize && file.size >= minSize
-}
+export const fileMatchSize = (file, maxSize, minSize) =>
```

Comment:
Let’s revert this file since I don’t see any reason for it to change. Especially not in this PR